typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -1,4 +1,22 @@
-import { ChainMetadata, CompleteChainMap } from './types';
+import { ChainName } from '../types';
+
+/**
+ * A Domain (and its characteristics)
```

Comment:
Should this be Chain?
```suggestion
 * A Chain (and its characteristics)
```